source/usage-examples/find.txt

```diff
@@ -3,3 +3,54 @@ Find Multiple Documents
 =======================
 
 .. default-domain:: mongodb
+
+You can find multiple documents in a collection by using the ``Find()``
+method.
+
+The following example passes a query filter to the ``Find()`` method,
+which matches documents in the ``zips`` collection if the value of their
```

Comment:
```suggestion
The following example passes a query filter to the ``Find()`` method
that matches documents in the ``zips`` collection if the value of their
```

Nit: I think this should be "that" as I think the contents of the [query is essential information in the context of the page](https://www.grammarly.com/blog/which-vs-that/) (it is necessary for understanding expected results section).

Reply:
Leaving it as is to maintain consistency with other pages. Since there is so little information preceding the code snippet, I don't think people will skip that part of the sentence even if it starts with a "which"

Reply:
I don't think in this case it is worth emulating other examples, as the project is still really new (not much precedent). I think this change is inline with the [usage example template](https://docs.google.com/document/d/1hCXhDzcIXo3oqs9OzH158v98lMwIaJFkReln1y-TXjw/edit#), so I think it is better to get the wording as correct as possible rather than try and make the content as close to other content as possible.